### PR TITLE
feat(node): dns.promises module add getDefaultResultOrder api

### PR DESCRIPTION
**Pull request by btea**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://nodejs.org/dist/latest-v21.x/docs/api/dns.html#dnspromisesgetdefaultresultorder

### CODE REVIEW

**Review by Semigradsky — CHANGES_REQUESTED** [review]
Please add these changes to node/ts4.8 folder too

**Review by Semigradsky — APPROVED** [review]

**Comment by Semigradsky** [comment]
Ah, I missed that in doc that was added to `v18.17.0` too.
Would be nice to add to `node/v18` and `node/v18/ts4.8`  folders too.

**Comment by btea** [comment]
@Semigradsky Thanks for the reminder, I've updated it.

File: types/node/v18/dns/promises.d.ts [hunk 1]
```diff
@@ -307,6 +307,14 @@ declare module "dns/promises" {
      * @since v10.6.0
      */
     function reverse(ip: string): Promise<string[]>;
+    /**
+     * Get the default value for `verbatim` in {@link lookup} and `dnsPromises.lookup()`. The value could be:
+     *
+     * * `ipv4first`: for `verbatim` defaulting to `false`.
+     * * `verbatim`: for `verbatim` defaulting to `true`.
+     * @since v20.1.0
```
Comment (Semigradsky) [inline types/node/v18/dns/promises.d.ts:315]: Here should be `* @since v18.17.0`

File: types/node/v18/ts4.8/dns/promises.d.ts [hunk 2]
```diff
@@ -307,6 +307,14 @@ declare module "dns/promises" {
      * @since v10.6.0
      */
     function reverse(ip: string): Promise<string[]>;
+    /**
+     * Get the default value for `verbatim` in {@link lookup} and `dnsPromises.lookup()`. The value could be:
+     *
+     * * `ipv4first`: for `verbatim` defaulting to `false`.
+     * * `verbatim`: for `verbatim` defaulting to `true`.
+     * @since v20.1.0
```
Comment (Semigradsky) [inline types/node/v18/ts4.8/dns/promises.d.ts:315]: ditto

File: types/node/v18/dns/promises.d.ts (hunk 1 above)
Comment (btea) [reply types/node/v18/dns/promises.d.ts]: Done.

**Review by Semigradsky — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]
@btea thanks!

**Comment by btea** [comment]
Ready to merge

**Pull request merged**